### PR TITLE
updated app to latest uxn from submodule

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: app/Emulator/Platform/app_uxn.c [hunk 1]
```diff
@@ -4,17 +4,16 @@
 
 #import "uxn.h"
 
-#include "devices/apu.c"
-#include "devices/ppu.c"
+#include "devices/audio.c"
```
Comment (craigcomstock) [inline app/Emulator/Platform/app_uxn.c:8]:
```
commit c3506869b50287f6e21c1c8f4ce4479a501f5e06
Author: neauoire <aliceffekt@gmail.com>
Date:   Tue Dec 28 13:47:35 2021 -0800

    Renamed Apu to Audio

commit 7b33cf3d9554bbca8bd8fb4f5a1b0976a5b8a6dd
Author: neauoire <aliceffekt@gmail.com>
Date:   Tue Dec 28 13:37:26 2021 -0800

    Renamed PPU to Screen
```

**Comment by paiv** [comment]
Hi @craigcomstock , there will be no submodules in this repo, sorry if this was unexpected. Feel free to detach your fork, it's all MIT license.

**Comment by craigcomstock** [comment]
> Hi @craigcomstock , there will be no submodules in this repo, sorry if this was unexpected. Feel free to detach your fork, it's all MIT license.

Ah, no problem. I'd rather contribute an updated uxn than stick to submodules. Not that big of a change.

You prefer to not use submodules then?

**Comment by paiv** [comment]
@craigcomstock Yes, thank you.

**Comment by craigcomstock** [comment]
made a new PR with no submodule :+1: https://github.com/paiv/uxn-ios/pull/2

**Pull request closed without merging**